ush/soca/run_jjobs.py

```diff
@@ -135,18 +135,14 @@ def copy_bkgs(self):
         Fill the ROTDIR with backgrounds
         TODO: replace by fill comrot?
         """
-        file_descriptor = components_short[self.component]+'f0'
-        self.f.write("mkdir -p ${ROTDIR}/${PSLOT}/gdas.${PDY}/${gcyc}/${COMPONENT}/\n")
-        command = "cp -r ${COMIN_GES}/*." + file_descriptor + "*.nc"
-        command += "  ${ROTDIR}/${PSLOT}/gdas.${PDY}/${gcyc}/${COMPONENT}/\n"
-        self.f.write(command)
-        # Special case for the ocean: DA for ice & ocean
-        if self.component == 'ocean':
-            # staging ice backgrounds
-            ice_file_descriptor = components_short['ice']
-            self.f.write("mkdir -p ${ROTDIR}/${PSLOT}/gdas.${PDY}/${gcyc}/ice/RESTART\n")
-            self.f.write("cp -r ${COMIN_GES}/../ice/*icef0*.nc ${ROTDIR}/${PSLOT}/gdas.${PDY}/${gcyc}/ice/\n")
-            self.f.write("cp -r ${COMIN_GES}/../ice/RESTART/*cice_model.res.nc ${ROTDIR}/${PSLOT}/gdas.${PDY}/${gcyc}/ice/RESTART/\n")
+        self.f.write("mkdir -p ${COM_OCEAN_HISTORY}/\n")
```

Comment:
these templates have to be defined in the script I think, have you tested this?

Reply:
yes, it's tested in the ```ctests```. 
```run_jjobs.py``` runs the j-job, where the ```COM_*``` are defined.